examples/python/smpi_all2all/all2all.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2010-2024. The SimGrid Team. All rights reserved.
+#
```

Comment:
Your examples seem nice, but we need tesh files too. I could take care of it, but if you use random in your file, will your output remain nicely stable? I confess I did not read the examples in detail, so I'm sorry if your code is actually reproducible.

Reply:
I just need to add a seed to make them reproducible.

Reply:
That's true, that's a good approach.